changelog/v1.7.0-beta3/istio-discovery.yaml

```
@@ -0,0 +1,8 @@
+changelog:
+  - type: HELM
+    issueLink: https://github.com/solo-io/gloo/issues/3924
+    description: |
```

we probably want `>-` here
```suggestion
    description: >-
```